cmd/crank/foundation/util.go

```diff
@@ -0,0 +1,70 @@
+// Package foundation is for functions and helpers shared between components that can be exported to things like crossplane-contrib.
```

Comment:
nit: may be something like

```suggestion
// Package foundation provides functions and helpers shared between CLI utilities, available for import by external tools interacting with Crossplane.
```